### PR TITLE
Feature/aapd 36 dbstatus=confirmed

### PR DESCRIPTION
**Pull request by suityou01**
## Ticket Number

https://pins-ds.atlassian.net/browse/AAPD-36

## Description of change

Change status of LPA User from added to confirmed on enter-code

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [x] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/forms-web-app/src/controllers/common/enter-code.js [hunk 1]
```diff
@@ -319,6 +323,10 @@ const postEnterCodeLPA = (views) => {
 		if (!tokenVerification(res, token, views, id)) return;
 
 		try {
+			const currentUserStatus = getLPAUserStatus(id);
```
Comment (bryn-pins) [inline packages/forms-web-app/src/controllers/common/enter-code.js:326]: think there's a missing await here

File: packages/forms-web-app/src/services/lpa-user.service.js [hunk 2]
```diff
@@ -15,6 +15,16 @@ const { getUserById } = require('../lib/appeals-api-wrapper');
  * @property {string} lpaCode - If code of the lpa the user belongs to.
  */
 
+/**
+ * Creates the user object within the session object of the request for the successfully logged in LPAUser
```
Comment (bryn-pins) [inline packages/forms-web-app/src/services/lpa-user.service.js:19]: Think this comment needs updating

File: packages/forms-web-app/src/controllers/common/enter-code.js (hunk 1 above)
Comment (suityou01) [reply packages/forms-web-app/src/controllers/common/enter-code.js]: Good spot. Fixed unit test to mockResolvedValue also

File: packages/forms-web-app/src/services/lpa-user.service.js (hunk 2 above)
Comment (suityou01) [reply packages/forms-web-app/src/services/lpa-user.service.js:19]: To what?

**Review by bryn-pins — APPROVED** [review]

**Pull request merged**